Cleanup obsolete archiving and cleaning settings (#2479)
kyma-project/kyma-environment-broker: notable-changes/1.22.6/notable-change.md

<!--{"metadata":{"requirement":"RECOMMENDED","type":"INTERNAL","category":"CONFIGURATION","additionalFiles":0}}-->

# Updating Kyma Environment Broker: Removal of Archiving and Cleaning Flags

> [!NOTE] 
> This is a recommended change. The `archiving` and `cleaning` feature flags are now removed because both processes are always active. 

> > Without updating the configuration, obsolete settings will have no effect and may cause confusion.

Reviewer: ```suggestion
> Without updating the configuration, the obsolete settings have no effect, but may cause confusion.
```

## What's Changed

The following configuration flags have been permanently removed from KEB:

```yaml
archiving:
  enabled:
  dryRun:
cleaning:
  enabled:
  dryRun:
```

> Archiving and cleaning processes are now always active. `enabled` and `dryRun` settings are ignored and should be removed.

Reviewer: ```suggestion
The archiving and cleaning processes are now always active. The `enabled` and `dryRun` settings are ignored and should be removed.
```

## Procedure

1. Open the KEB configuration file.

> 2. Remove the following sections entirely if present:

Reviewer: ```suggestion
2. If the following sections are present in your configuration, remove them.
```
